bridges/otelslog/handler.go

```diff
@@ -239,6 +239,20 @@ func (h *Handler) convertRecord(r slog.Record) log.Record {
 	return record
 }
 
+// levelToText converts a slog.Level to a string representation of the severity range
+func levelToText(level slog.Level) string {
+	switch {
+	case level < slog.LevelInfo:
+		return "DEBUG"
+	case level < slog.LevelWarn:
+		return "INFO"
+	case level < slog.LevelError:
+		return "WARN"
+	default:
+		return "ERROR"
```

Comment:
I would add a check for the exact `LevelError` value, and default anything else to `UNKNOWN`.

```suggestion
	case level = slog.LevelError:
		return "ERROR"
	default:
		return "UNKNOWN"
```

Reply:
So the reason I did with matching ranges is that slog levels are ranges same as otel severity is. 

Take a look at the slog.Level string implementation: https://cs.opensource.google/go/go/+/refs/tags/go1.24.2:src/log/slog/level.go;l=68

So INFO+2 is a valid `slog.Level.String()` value which I intentionally mapped to "INFO", discarding the range offset. Same applies with other levels.

If we're on the same page regarding the slog level ranges and you have taken this into consideration, I am more than happy to accept your proposed changes

Reply:
If slog introduces a new log level, they will change that implementation right away.
We may miss that change for a bit of time, therefore sending the invalid level string. Hence the `unknown`.